### DISCUSSION TITLE
Module not found: Can't resolve '/img/logo.png

### DISCUSSION THREAD

**Discussion by awitherow** [vercel/next.js#17639] [Help] [answered]
I set up the static routing as I thought was expected according to the docs, but it does not work.

Do I have something set up wrong in my repo? 

I am on Nextjs 9.5.3 installed with the latest stable version of node.

The repo is here -> https://github.com/awitherow/firebase-starter

When I run `npm run dev` I get 

```./components/Layout/Layout.js:16:0
Module not found: Can't resolve '/img/logo.png'
  14 | 
  15 | import styles from "./Layout.module.scss";
> 16 | import logo from "/img/logo.png";
  17 | 
  18 | function Layout({ pageId, children, user, mainClass, location }) {
  19 |   const [showModal, setModalShown] = useState(false);
```

Even though in the documents it says that I should be able to reference directly from `public`.

Idk what to do...

**Comment by mxmtsk** [comment, marked as answer]
You can just reference the file in the public folder the "normal" way like so:

```
<img src="/img/logo.png" className={styles.logo} alt="logo" />
```

See here:
https://nextjs.org/docs/basic-features/static-file-serving

If you want to import the file, you need to use the relative path, like you do with your normal imports:

```
import logo from "../../public/img/logo.png";
```

**Comment by maimaonsare** [comment]
I believe you are getting the error because you are trying to reference an image from a css file. 
inside your css file, you can create the image url like below, and the error will disappear while you get the image:

background-image: url('../public/images/your-image.jpg');
